source/guides/stacks/dlrs-inference.rst

```diff
@@ -85,7 +85,7 @@ Although this guide assumes a |CL| host system, it has also been validated with
 Recommended Hardware
 ====================
 
-We validated this guide on an `Intel Cascade Lake`_ server and this is recommended to get optimal performance and take advantage of the built in  Intel® Deep Learning Boost functionality.
+We validated this guide on a server with a 2nd Generation Intel Xeon Scalable processor, formerly Cascade Lake, and this is recommended to get optimal performance and take advantage of the built in Intel® Deep Learning Boost (Intel® DL Boost) functionality.
```

Comment:
```suggestion
We validated this guide on a server with a 2nd Generation  Intel® Xeon® Scalable processor, formerly Cascade Lake, and this is recommended to get optimal performance and take advantage of the built in Intel® Deep Learning Boost (Intel® DL Boost) functionality.
```

Despite the fact that this doc references  "Intel® Xeon® platform", above, the "Intel® Xeon® Scalable processor" is a unique phrase whose meaning is slightly different. One is general; the other is more specific.

Reply:
I didn't add in the (r)s b/c this is second use of the trademark on the page (The trademark part is 'Intel® Xeon®' which is used as an adjective with one of the approved nouns - as I read this description from the DB, this is the second use of the trademark on the page... glad to discuss if needed :) ). 

I see I did accidentally remove the original link tho! Will add back in.

Reply:
@mvincerx Thanks for the detailed review! Applied updates - I didn't make all changes, based on current working knowledge of TM&B rules (from legal + DB). I explained my reasoning inline w/ your original feedback.